cli/src/deploy/mod.rs

```diff
@@ -386,85 +404,65 @@ pub async fn info() {
   };
   clusters.columns.insert(4, column);
 
-  let mut deploy_names = HashSet::new();
-  let mut data: Vec<Vec<&dyn Display>> = vec![];
+  let mut app_data: Vec<Vec<&dyn Display>> = vec![];
+  let mut profile_data: Vec<Vec<&dyn Display>> = vec![];
+  let mut deploy_profiles = HashSet::new();
   for app in &mut apps {
-    deploy_names.insert(&app.deployName);
-    data.push(vec![
+    app_data.push(vec![
       &app.id,
       &app.url,
       &app.deployName,
       &app.size,
       &app.status,
     ]);
+    for (i, profile) in app.cloudConfigs.iter().enumerate() {
+      if deploy_profiles.contains(&app.deployName) {
+        continue;
+      }
+      if i == 0 {
+        profile_data.push(vec![&app.deployName, &profile.region, &profile.vmType])
+      } else {
+        profile_data.push(vec![&"", &profile.region, &profile.vmType])
```

Comment:
Why empty string here?

Reply:
If a deploy profile has more than one cloud config (multi-region or multi-cloud) then this let's us show it within the same row under that deploy profile in the table